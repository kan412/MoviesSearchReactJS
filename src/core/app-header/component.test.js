import React from 'react';
import { shallow } from 'enzyme';
import AppHeader from './component';

describe('<AppHeader />', () => {
  it('render without breaking', () => {
    const component = shallow(<AppHeader />);
    expect(component).toMatchSnapshot();
  });
});
