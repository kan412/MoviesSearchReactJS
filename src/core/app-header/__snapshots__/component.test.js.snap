// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppHeader /> render without breaking 1`] = `
<header
  className="header"
>
  <div
    className="inner"
  >
    <a
      href="/"
    >
      <h1
        className="logo"
      >
        Netflix
        <span>
          roulette
        </span>
      </h1>
    </a>
  </div>
</header>
`;
